site/examples/inputs.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { HtmlExample } from '../html_example';
+
+const inputWithButton = `<fieldset class='relative'>
+  <button class='btn absolute top right w96 round-r'>Search</button>
+  <input class='input pr96' placeholder='Search'>
```

Comment:
The input needs some additional padding on the right, so text doesn't jam up against the button.